plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -79,8 +79,10 @@ class ServiceNetworkingConfigSection extends ServiceConfigBaseSectionDisplay {
                 prop: keys.protocol,
                 className: getColumnClassNameFn(),
                 render(prop, row) {
-                  return getDisplayValue(row[prop])
-                    .split(',').join(', ');
+                  let protocol = row[prop] || '';
+                  protocol = protocol.replace(',', ', ');
```

Comment:
I would still stick to `/,(?=\S)/g`, because it could always change and then we need to find those locations where we did not think about having more then two items.